src/lib/configs/project/provider.ts

```diff
@@ -431,11 +449,19 @@ export async function initNewReadonlyProviderConfig({
   return providerConfig;
 }
 
-export function initProviderConfig() {
+export function initProviderConfig(name?: string | undefined) {
+  if (name !== undefined) {
+    setProviderConfigName(name);
+  }
+
   return getConfigInitFunction(getInitConfigOptions())();
 }
 
-export function initReadonlyProviderConfig() {
+export function initReadonlyProviderConfig(name?: string | undefined) {
```

Comment:
```suggestion
export function initReadonlyProviderConfig(name?: string) {
```

Looks like in js omitting argument is the same as passing `undefined` directly

Reply:
Yeah for function params this seems to be true, will apply this imporvement. This is not true for objects though, cause for objects a new key might be created with undefined inside, which would a different runtime behaviour